app/src/main/java/com/alphawallet/app/service/TokensService.java

```diff
@@ -105,12 +104,16 @@
     private Disposable openSeaQueryDisposable;
 
     private static boolean done = false;
+    CustomSettings customSettings;
 
```

Comment:
Why not import the CustomSettings directly from DI? This way you don't need context